test/api.js

```diff
@@ -249,6 +233,18 @@ test('test file that immediately exits with 0 exit code ', function (t) {
 		});
 });
 
+test('testing nonexistent files rejects', function (t) {
+	t.plan(2);
+
+	var api = new Api([path.join(__dirname, 'fixture/broken.js')]);
+
+	api.run()
+		.catch(function (err) {
+			t.ok(err);
+			t.true(/Couldn't find any files to test/.test(err.message));
```

Comment:
use taps `t.match(string, regexp)` in all new tests. It gives us better error messages. See #256